marklogic-data-hub/src/test/ml-config/databases/final-database.json

```diff
@@ -18,6 +18,83 @@
       "collation" : "http://marklogic.com/collation//S1",
       "range-value-positions" : false,
       "invalid-values" : "reject"
+    },
+    {
+      "scalar-type": "decimal",
+      "namespace-uri": "",
+      "localname": "decimalProp",
+      "collation": "",
+      "range-value-positions": false,
+      "invalid-values": "reject"
+    },
+    {
+      "scalar-type": "double",
+      "namespace-uri": "",
+      "localname": "doubleProp",
+      "collation": "",
+      "range-value-positions": false,
+      "invalid-values": "reject"
+    },
+    {
+      "scalar-type": "float",
+      "namespace-uri": "",
+      "localname": "floatProp",
+      "collation": "",
+      "range-value-positions": false,
+      "invalid-values": "reject"
+    },
+    {
+      "scalar-type" : "string",
+      "namespace-uri" : "",
+      "localname" : "strNameProp",
+      "collation" : "http://marklogic.com/collation/codepoint",
+      "range-value-positions" : false,
+      "invalid-values" : "reject"
+    },
+    {
+      "scalar-type" : "string",
+      "namespace-uri" : "",
+      "localname" : "srchEntyProp1",
```

Comment:
Let's not omit any letters in words, there's no benefit in doing that - just do "searchEntityProp1".